app/views/archive_faqs/manage.html.erb

```diff
@@ -4,6 +4,12 @@
 
 <!--subnav-->
 <%= render :partial => 'admin/admin_nav' %>
+<ul class="navigation actions">
+  <% if Globalize.locale.to_s == "en" && policy(ArchiveFaq).full_access? %>
+    <li><%= link_to t(".new_faq_category"), new_archive_faq_path %></li>
+    <li><%= link_to t(".reorder_faqs"), manage_archive_faqs_path(language_id: params[:language_id]) %></li>
+  <% end %>
+</ul>
```

Comment:
All of the navigation items should be in the same `<ul>`. (The existing code in the wrangling guidelines admin index that puts them in separate lists is wrong.)

Also, as a general rule, we don't want to include empty elements on a page. So logic like `<% if Globalize.locale.to_s == "en" && policy(ArchiveFaq).full_access? %>` would go outside the `<ul>` in this code, since there are no `<li>` elements that would be included if that logic were `false`.